Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.AdvancedDataGridView/Controls Toolkit/KryptonAdvancedDataGridView.cs

```diff
@@ -716,7 +716,7 @@ public bool FilterAndSortEnabled
         {
             get
             {
-                return (!String.IsNullOrEmpty(_sortString) ? _sortString : "");
+                return !String.IsNullOrEmpty(_sortString) ? _sortString : "";
```

Comment:
PLease use `string.Empty` so that it does not have build an instance of an empty string each time.
Also it shows that this is intended and not just some accidentally emptied string

Reply:
Done